cardano-cli/test/Test/CLI/Shelley/Golden/Address/Build.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Are we using `ScopedTypeVariables` in this module?

```suggestion
```

Reply:
Done